src/redis-cli.c

```diff
@@ -1345,6 +1345,7 @@ static sds cliFormatReplyTTY(redisReply *r, char *prefix) {
             char numsep;
             if (r->type == REDIS_REPLY_SET) numsep = '~';
             else if (r->type == REDIS_REPLY_MAP) numsep = '#';
+            else if (r->type == REDIS_REPLY_PUSH) numsep = '>';
```

Comment:
aren't we introducing a breaking change (for scripts that work with redis-cli)?
i thought to do that only in interactive mode, but actually that would be confusing (someone experimenting with interactive in order to know how to write his script).
@yossigo please advise.

Reply:
This output is only for TTYs, so script shouldn't be affected unless they're trying to simulate a TTY (like we do in the TTY tests).

Reply:
It seems `redis-cli --no-raw` in scripts is affected, so it's breaking change for this use case.

Reply:
All other types have their own prefix, so it's odd that push doesn't. Push shares the prefix with array (`1)`, `2)` etc.).

OTOH, this is not coupled with this PR so I can move it to a separate PR.

Reply:
I spoke to Yossi, let's drop this `>` change, and add it in 8.0